scripts/process-docusaurus-v3-upgrade.js

```diff
@@ -0,0 +1,277 @@
+const replace = require('replace')
+const fs = require('fs').promises
+const path = require('path')
+const { join } = path
+
+async function editLine(file, lineNumber, oldContent, newContent) {
+  const data = await fs.readFile(file, 'utf8')
```

Comment:
I had made the edits line by line so that it would be easy for anyone to identify what items were changed..but we could do  a simple regex check in specific files ..cause I noticed that its only a few files that require changing so we could check only in those files.

Reply:
@Eomm as for the latest branch I also noticed the issue where we'd be updating beyond the scope. To ensure this I believe it might require some steps

1. Merge the changes with the latest branch still being checked
2. Add a docusaurus v3 check in the fastify/fastify build to ensure the latest is always docusaurus compatiple
```
 npx docusaurus-mdx-checker
```
3. Remove the latest check in the docusaurus v3 upgrade